understanding-text-format/wasm-table.wat

```diff
@@ -8,5 +8,4 @@
   (type $return_i32 (func (result i32)))
   (func (export "callByIndex") (param $i i32) (result i32)
     local.get $i
-    call_indirect (type $return_i32))
-)
+    call_indirect (type $return_i32)))
```

Comment:
I think in examples like these, it's clearer to keep the closing paren on its own line.

Reply:
Yeah, I think we're a bit irregular here because: putting *every* closing paren on its own line ends up being too noisy whereas maximally putting parens on the same line loses visual structure.  My personal style (which I won't claim is the best, but I like) is to always put `func` and `module` closing parens on their own line, and everything else that is "smaller" gets to share a line.  WDYT?